src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -3,6 +3,71 @@
 @import '../../styles/mixins/overlay';
 @import '../../components/pill-selector-dropdown/mixins';
 
+.is-responsive-web {
+    .collaborator-list {
+        min-width: 0;
+    }
+
+    @include breakpoint($medium-screen) {
+        .usm-menu-description {
+            white-space: normal;
+        }
+
+        .usm-share-access-menu {
+            width: min(400px, 90%);
+        }
+    }
+
+    @include breakpoint($small-medium-screen) {
+        .collaborator-list {
+            height: auto;
+            border: 0;
+            border-radius: 0;
+            box-shadow: none;
+        }
+
+        .shared-link-field-container {
+            align-items: center;
+            height: 34px;
+
+            .btn {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                width: 60px;
+                min-width: 0;
+                height: 34px;
+            }
+        }
+
+        .email-shared-link-btn {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            width: 34px;
+            height: 34px;
```

Comment:
Since this component also has class `btn`, I think most of these styles should come from this block: https://github.com/box/box-ui-elements/blob/75e306077e3b8e3c8590bbac5ee47b4fd796d6bf/src/features/unified-share-modal/UnifiedShareModal.scss#L33-L40

Reply:
Removed redundant styles.